---
bool success
float64 speed_of_sound
bool acoustic_enabled
bool dark_mode_enabled
float64 mounting_rotation_offset
string range_mode